powers.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+/*The purpose is to find all the values of 2^n under 1000. Jack Rosen*/
+int main()
+{
+	int product = 1;
+	printf("%d\n",product);
+	while (product <= 500)
```

Comment:
So with that said, while you got the assignment done, it wouldn't be sustainable if we changed n